Source/WebCore/bindings/scripts/test/TestCallbackOrDict.idl

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2024 Keith Cirkel <webkit@keithcirkel.co.uk>. All rights reserved.
+ * Copyright (C) 2024 Apple Inc. All rights reserved.
```

Comment:
I don't think you should be adding Apple Inc. unless this was taken from elsewhere? (In which case it should appear first.)